web-ui/dune

```diff
@@ -2,6 +2,7 @@
  (target static.ml)
  (deps
   %{workspace_root}/web-ui/static/css/main.css
+  %{workspace_root}/web-ui/static/js/step-logs.js
```

Comment:
This was missing and was causing a 404 on `step-logs.js` when I deployed this last week.